recipes/CVSS/S2ST/utils/embedding.py

```diff
@@ -0,0 +1,24 @@
+import json
```

Comment:
Having utils in our recipe is unusual. Please add a big HEADER to explain what this is, and what it uses and how it does it.

Reply:
Also, I feel like instead of having a file on the side, we could simply have this class on the train.py as it is specific to this recipe? @Adel-Moumen @mhn226 @Gastron @pplantinga any point of view on this design aspect?

Reply:
Thank you for the feedback. Currently, only the LJSpeech S2ST recipe has been refactored. My plan is to remove the custom embedding loader and store each utterance in a separate .npy file, similar to the LJSpeech S2ST recipe.